tests/core/test_edit_package_properties.py

```diff
@@ -63,7 +83,21 @@ def test_throws_error_if_current_package_properties_are_malformed(tmp_path, prop
         edit_package_properties(path, properties)
 
 
-def test_throws_error_if_new_package_properties_are_incorrect(properties_path):
-    """Should throw NotPropertiesError if the new package properties are incorrect."""
+def test_adds_custom_fields(
+    properties_path,
+):
+    """Should add custom fields to properties."""
+    # Given
+    current_properties = read_json(properties_path)
+    new_properties = {"custom-field": "custom-value"}
+
+    # When, Then
+    edit_package_properties(
+        properties_path, new_properties
+    ) == current_properties | new_properties
```

Comment:
```suggestion
    assert edit_package_properties(
        properties_path, new_properties
    ) == current_properties | new_properties
```

Was this assert missing?